<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,3,0,1" columnstretch="0,1,1">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0" rowspan="4">
       <widget class="QLabel" name="floorsLabel">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0.5px solid grey;
background-color: rgb(119, 118, 123);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;FLOORS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="passengerViewLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0.5px solid grey;
border-left: none;
background-color: rgb(119, 118, 123);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;PASSENGER VIEW&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="consoleLogsLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(119, 118, 123);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; writing-direction:vertical-lr&quot;&gt;CONSOLE LOGS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="adminViewLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(119, 118, 123);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ADMIN VIEW&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="elevatorsLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0.5px solid grey;
border-right: none;
background-color: rgb(119, 118, 123);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ELEVATORS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="addPassengerPushButton">
          <property name="text">
           <string>Add Passenger</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletePassengerPushButton">
          <property name="text">
           <string>Delete Passenger</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="2">
       <widget class="QListWidget" name="consoleLogsListWidget"/>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QGridLayout" name="adminGrid" rowstretch="0,0,0,0" columnstretch="4,1,4">
        <item row="1" column="0">
         <widget class="QLabel" name="numFloorsLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string># Floors</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="mapSettingsLabel">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;MAP SETTINGS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="buildingEmergencyTestingLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;BUILDING EMERGENCY TESTING&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="changeDimensionsButton">
          <property name="text">
           <string>Change Dimensions</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="numElevatorsLabel">
          <property name="text">
           <string># Elevators</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="numberElevatorsSpinBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="numberFloorsSpinBox"/>
        </item>
        <item row="1" column="2" rowspan="3">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="firePushButton">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>Fire</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="powerOutagePushButton">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>Power Outage</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="passengerLayout" stretch="0,3">
        <item>
         <widget class="QLabel" name="passengerLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Passenger:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="passengerComboBox"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1" rowspan="3">
       <layout class="QGridLayout" name="elevatorGrid"/>
      </item>
      <item row="3" column="2">
       <layout class="QGridLayout" name="passengerViewLayout" columnstretch="0,0,0,1,2">
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="6" column="1">
         <widget class="QLineEdit" name="maxWeightElevatorInput"/>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="elevatorTestingLabel">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ELEVATOR TESTING&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="weightElevatorInput"/>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QPushButton" name="blockDoorPushButton">
          <property name="text">
           <string>Block Door</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="5">
         <widget class="Line" name="line">
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2" rowspan="7">
         <widget class="Line" name="line_2">
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="startingFloorSpinBox"/>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QPushButton" name="downFloorPushButton">
          <property name="text">
           <string>Down</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="startingFloorLabel">
          <property name="text">
           <string>Starting Floor</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="floorPanelLabel">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;FLOOR PANEL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="2">
         <widget class="QPushButton" name="upFloorPushButton">
          <property name="text">
           <string>Up</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" rowspan="2" colspan="2">
         <layout class="QGridLayout" name="floorHelpButtonGrid">
          <item row="0" column="0">
           <widget class="QPushButton" name="helpPushButton">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color:white; background-color:red;font-weight:bold;</string>
            </property>
            <property name="text">
             <string>Help</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="3" colspan="2">
         <widget class="QLabel" name="systemDisplayLabel">
          <property name="styleSheet">
           <string notr="true">border: 1px solid black;
background-color: rgb(119, 118, 123);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Elevator Display&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="closePushButton">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="openPushButton">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="maxElevatorWeightLabel">
          <property name="text">
           <string>Max elevator weight: </string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="elevatorWeightLabel">
          <property name="text">
           <string>Weight:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3" rowspan="6" colspan="2">
         <layout class="QGridLayout" name="elevatorDisplayGrid"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
